tests/test_utils.py

```diff
@@ -0,0 +1,311 @@
+from ubi_manifest.worker.tasks.depsolver.utils import (
+    vercmp_sort,
+    _keep_n_latest_rpms,
+    flatten_list_of_sets,
+    parse_bool_deps,
+    get_n_latest_from_content,
+    create_or_criteria,
+)
+from ubi_manifest.worker.tasks.depsolver.models import UbiUnit
+from pubtools.pulplib import RpmUnit, Criteria
+import pytest
```

Comment:
Minor suggestion: I think the import order is getting a bit messy here and PEP8-noncompliant.  You might consider adding isort into .pre-commit-config.yaml in this repo as it will be able to automatically keep them clean.

```yaml
-   repo: https://github.com/pycqa/isort/
    rev: 5.6.4
    hooks:
    - id: isort
```